docs/api/users-update-by-id.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+---
+
+# Update user by ID
+
+Allows the editing of [`user`](user) objects that already exist. You will need to know the `id` parameter of the user you want to edit.
+
+> [!IMPORTANT]
```

Comment:
This is a good (and important) note, but it might not be in the best location.

Because this has to do with the request body, maybe put it down in that section. Remember that reference topics are rarely read sequentially.

Reply:
Done!